pages/common/fping.md

```diff
@@ -0,0 +1,15 @@
+# fping
+
+> A more powerful ping which ping multiple hosts
+
+- List alive host within subnet via netmask
+
+`fping -a -g 192.168.1.0/24`
+
+- List alive host within subnet via ip range
```

Comment:
Pluralize "host" here as well, and make "ip" uppercase. "Via" could also be replaced by "using a".